netaddr.go

```diff
@@ -406,6 +406,14 @@ func (ip IP) v6(i uint8) uint8 {
 	}
 }
 
+func (ip IP) v6_16(i uint8) uint16 {
```

Comment:
if you change `_` to `u` I think it'll still okay and not have an underscore. And be a bit more informative?